documentation/general/design/dash-holistic-view.md

```diff
@@ -0,0 +1,618 @@
+- **DASH programmable hardware**. With the help of network hardware technology
+providers, we are collaborating to create an open forum capitalizing on the use
+of programmable hardware including **SmartNICs**, **SmartSwitches**, and
+**SmartAppliances**. The documentation related to these devices will be made
+available by the providers themselves.  
+- **Scenarios**. Cover DASH extending SONiC APIs and related comprehensive set
+of objects to describe networking services for the cloud. The 7 initial services
+(in draft) describe this in more detail. We will also work with cloud and
+enterprise providers to further extend SONiC DASH to meet their specific needs.
+For more information,see the [Scenarios](#scenarios) section.
+- **Conformance and performance test suite**. A suite of tests to validate that
+  the DASH devices satisfy the standard DASH conformance and performance
+  requirements. For more information about test documentation, test scripts,
+  test configurations and other artifacts required to test a DASH device or
+  devices, see [Performance
+  testing](https://github.com/Azure/DASH/tree/main/test). 
+
+## Logical architecture (software stack) 
+
+ DASH builds upon the traditional SONiC architecture, which is documented in the
+ SONiC Wiki under [SONiC System
+ Architecture](https://github.com/Azure/SONiC/wiki/Architecture#sonic-system-architecture).
+ The following descriptions assume familiarity with the SONiC architecture and
+ will describe DASH as incremental changes relative to traditional SONiC. Notice
+ that DASH adds a new **SDN control plane** via **gNMI** in the **DASH
+ container**. The following figure depicts the DASH software stack.
+
+![dash-software-stack](./images/hld/dash-hld-software-stack.svg)
+
+<figcaption><i>Figure 2 - DASH software stack</i></figcaption><br/>
+
+#### SDN controller
+
+The SDN controller is **primarily responsible for controlling the DASH overlay
+services**, while the traditional SONiC application containers are used to
+manage the underlay (L3 routing) and hardware platform. Both the DASH container
+and the traditional SONiC application containers sit atop the Switch State
+services (SWSS) layer, and manipulate the Redis application-layer DBs; these in
+turn are translated into SAI dataplane obects via the normal SONiC orchestration
+daemons inside SWSS.
+
+The SDN controller controls the overlay built on top of the physical layer
+(underlay) of the infrastructure.  From the point of view of the SDN control
+plane, when a customer creates an operation from the cloud portal (for example a
+VNET creation), the controller allocates the resources, placement management,
+capacity, etc. via the  **NorthBound interface APIs**.
+
+##### SDN and DPU High-Availability (HA)
+
+High Availability (HA) is important for hardware failover incidents. The SDN controller selects a pair of cards and
+configures them identically.  The only requirement on the card from the HA
```

Comment:
This description is obsolete - see the HA design docs. The SDN controller will set up the L3 routes through the ToR switch levels, possibly even T2 levels for a third backup appliance or smart ToR and configure the HA "Peers." HA sync protocols are WIP and will be standardized and interoperable, not vendor-specified.

Reply:
Verify the update doc.